packages/node-image-compute/base.packages

```diff
@@ -1,9 +1,110 @@
-# Packages common to all mediums.
+# Packages for any CN artifacts.
 # Format:
 #   package_name=version
 # The version is the same version reported by the OS package manager (e.g. zypper).
-cfs-state-reporter=1.9.0-1
-cfs-trust=1.6.0-1
-cray-switchboard=1.2.3-1
-cray-uai-util=1.0.13-1
-craycli=0.64.0-1
+
+# Analytics Base SLES Packages
+java-11-openjdk-devel
+
+# CDST GUI Packages
+libGLU1
+libatk-1_0-0
+libcairo-gobject2
+libcairo2
+libgcrypt20
+libgdk_pixbuf-2_0-0
+libgio-2_0-0
+libglvnd
+# libgpg-error # 'libgpg-error0' providing 'libgpg-error' is already installed.
+libgtk-3-0
+libgtksourceview-3_0-1
+libopenblas_pthreads0
+# librsvg2 # 'librsvg-2-2' providing 'librsvg2' is already installed.
+munge
+typelib-1_0-Gtk-3_0
+
+# COS General Packages
+cray-atom-energy-plugin
+cray-boot-parameters-shasta-compute
```

Comment:
Provides deps for BSS to create boot parameters, might be overzealous for our context.

Reply:
Removing for now.